presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -351,6 +355,88 @@ private PlanBuilder project(PlanBuilder subPlan, Iterable<Expression> expression
                 analysis.getParameters());
     }
 
+    private PlanBuilder project(
```

Comment:
I put it with the remaining ``project`` methods, and named it because it is adding projections for the aggregate arguments. Does that make sense, please?